### DISCUSSION TITLE
Terminology in English documentation

### DISCUSSION THREAD

**Discussion by johannadm** [netwerk-digitaal-erfgoed/solid-crs#557] [Project Erfgoedpod]
Hi all, 

Since there is no standard terminology for some concepts, I think it's best to define the terminology that will be used in the project.

To begin with I'd like your opinion on the next concepts:
1. 'Bruikleen'  - Is 'loan' a correct translation?
2. Issuing heritage institution / issuer - this could cause confusion since we use this word for referring to the identity provider
3. Requesting heritage institution / requester 
4. 'Uitlenen', 'ontlenen/lenen'  - which translations will we be using for the two directions?

**Comment by mielvds** [comment]
1. 'loan' seems to be the common term, a.o. [KBR](https://www.kbr.be/en/loans/) and [Libary of Congress](https://www.loc.gov/exhibits/loan/) use the term. Requesting one is known as a 'loan request'
2. hard to say, because agreements and such are always written by the institution and they use their own name. But in analogy to 3, perhaps the 'Lender'
3. often referred to as the 'Borrower', 'Borrowing Institution' or 'loan requester'
4. ontlenen = 'borrowing' and uitlenen = 'lending'

**Comment by johannadm** [comment]
We discussed in today's meeting that the translation for a 'bruikleencollectie' will be a 'loan collection'
